content/blog/2022/06/v1-release.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Kubewarden reaches 1.0.0 release 🎉
+authors:
+- Flavio Castelli
+date: 2022-06-08
+---
+
+Today, a year and a half since its conception, we're thrilled to announce
+the release of Kubewarden v1.0.0! 🎊 🥳
+
+Let's take a quick look at the major achievements of the project to see what `v1`
+brings both to Policy Authors and Operators.
+
+## Writing policies
+
+Kubewarden v1 allows Policy Authors to write policies using either Go, Rust, or Swift.
```

Comment:
```suggestion
Kubewarden v1 allows Policy Authors to write policies using any programming language that can generate a WebAssembly module. Such as using Go, Rust, or Swift. More information is available [here](https://webassembly.org/getting-started/developers-guide/). 
```

I think, it's an important element to mention

Reply:
This is true, but we have SDKs only for these languages